test/test_evaluators.py

```diff
@@ -7,10 +7,69 @@
 from vision_evaluation.evaluators import AveragePrecisionEvaluator, F1ScoreEvaluator, TopKAccuracyEvaluator, ThresholdAccuracyEvaluator, MeanAveragePrecisionEvaluatorForSingleIOU, EceLossEvaluator, \
     PrecisionEvaluator, RecallEvaluator, TagWiseAccuracyEvaluator, TagWiseAveragePrecisionEvaluator, MeanAveragePrecisionNPointsEvaluator, BalancedAccuracyScoreEvaluator, \
     CocoMeanAveragePrecisionEvaluator, BleuScoreEvaluator, METEORScoreEvaluator, ROUGELScoreEvaluator, CIDErScoreEvaluator, SPICEScoreEvaluator, RocAucEvaluator, MeanIOUEvaluator, \
-    ForegroundIOUEvaluator, BoundaryMeanIOUEvaluator, BoundaryForegroundIOUEvaluator, L1ErrorEvaluator
+    ForegroundIOUEvaluator, BoundaryMeanIOUEvaluator, BoundaryForegroundIOUEvaluator, L1ErrorEvaluator, GroupWiseEvaluator
 from vision_evaluation.prediction_filters import TopKPredictionFilter, ThresholdPredictionFilter
 
 
+class TestGroupWiseEvaluator(unittest.TestCase):
+    def test_group_wise_classification_evaluator(self):
+
+        gts = [{"targets": np.array([0, 0, 0, 0, 2, 2]), "groups": np.array([0, 0, 1, 1, 0, 0])},
+               {"targets": np.array([0, 0, 0, 1, 1, 1]), "groups": np.array([0, 0, 0, 1, 1, 1])}]
+
+        preds = [np.array([[0.8, 0.1, 0.1],
+                           [0.8, 0.1, 0.1],
+                           [0.1, 0.8, 0.1],
+                           [0.1, 0.8, 0.1],
+                           [0.1, 0.1, 0.8],
+                           [0.1, 0.1, 0.8]]),
+                 np.array([[0.8, 0.1],
+                           [0.1, 0.8],
+                           [0.8, 0.1],
+                           [0.1, 0.8],
+                           [0.8, 0.1],
+                           [0.1, 0.8]])]
+
+        n_group_glasses = [2, 2]
```

Comment:
I think I made a type, it should be n_group_classes

Reply:
resolved.